Sources/MongoSwift/ConnectionString.swift

```diff
@@ -33,6 +33,12 @@ internal class ConnectionString {
             mongoc_uri_set_option_as_bool(self._uri, MONGOC_URI_RETRYREADS, rr)
         }
 
+        // Per SDAM spec: If the ``directConnection`` option is not specified, newly developed drivers MUST behave as
+        // if it was specified with the false value.
+        if !self.hasOption("directConnection") {
```

Comment:
We should probably have an equivalent option in `MongoClientOptions` so that we can document its behavior

Reply:
done; also reworked the test to check all possible variations of unset / set via struct / set via URI